src/Generator.Rewrite/Options.cs

```diff
@@ -9,31 +9,31 @@ namespace OpenTK.Rewrite
     public class Options
     {
         /// <summary>
-        /// Set the path to the target assembly that should be rewritten.
+        /// Gets or sets the path to the target assembly that should be rewritten.
         /// </summary>
         [Option('a', "assembly", Required = true,
             HelpText = "The path to the target assembly that should be rewritten.")]
         public string TargetAssembly { get; set; }
 
         /// <summary>
-        /// Set the path to the strong name key which should be used to sign or resign the assembly.
+        /// Get or sets the path to the strong name key which should be used to sign or resign the assembly.
         /// </summary>
         [Option('k', "signing-key",
             HelpText = "The path to the strong name key which should be used to sign or resign the assembly.")]
         public string StrongNameKey { get; set; }
 
         /// <summary>
-        /// Enable calls to GL.GetError(), wrapped around each native call.
+        /// Gets or sets a value indicating whether enable calls to GL.GetError(), wrapped around each native call.
         /// </summary>
         [Option('d', "debug", Default = false,
             HelpText = "Enable calls to GL.GetError(), wrapped around each native call.")]
         public bool EnableDebugCalls { get; set; }
 
         /// <summary>
-        /// Force native calls to use DllImport instead of GetProcAddress.
+        /// Gets or sets a value indicating whether force native calls to use DllImport instead of GetProcAddress.
```

Comment:
Correct the grammar of this comment. It should be "whether native calls are fored to use".